Source/WebCore/svg/SVGFEDisplacementMapElement.cpp

```diff
@@ -122,6 +123,14 @@ void SVGFEDisplacementMapElement::svgAttributeChanged(const QualifiedName& attrN
     }
 }
 
+IntOutsets SVGFEDisplacementMapElement::outsets(const FloatRect& targetBoundingBox, SVGUnitTypes::SVGUnitType primitiveUnits) const
+{
+    auto halfScale = std::abs(scale() / 2);
+    auto maxDisplacement = FloatSize { halfScale, halfScale };
+    auto adjustedDisplacement = SVGFilterRenderer::calculateResolvedSize(maxDisplacement, targetBoundingBox, primitiveUnits);
+    return FEDisplacementMap::calculateOutsets(adjustedDisplacement);
```

Comment:
I think this function should pass the `resolvedScale` as is. `FEDisplacementMap::calculateOutsets()` should calculate the `maxDisplacement` or the `outsets` based on the sign of `resolvedScale / 2`